Cargo.toml

```diff
@@ -22,7 +22,7 @@ include = [
 winapi = { version = "0.3", features = ["handleapi", "winsock2", "ws2def", "ws2ipdef", "ws2tcpip"] }
 
 [target.'cfg(any(unix, target_os="wasi"))'.dependencies]
-libc = "0.2.54"
+libc = "0.2.146"
```

Comment:
Is there a change in 0.2.146 which is required for this support, or did you bump to latest just because it happens to be there?

Reply:
This requires a libc which supports QNX Neutrino which was added with v0.2.139: https://github.com/rust-lang/libc/releases/tag/0.2.139
I selected the most recent version instead.

Reply:
Since 0.2.139 is the actual requirement, and Cargo is free to choose a more recent version if desired, how about setting this to 0.2.139?